Mathlib/Order/Filter/Pointwise.lean

```diff
@@ -102,7 +102,7 @@ theorem principal_one : 𝓟 1 = (1 : Filter α) :=
 #align filter.principal_one Filter.principal_one
 #align filter.principal_zero Filter.principal_zero
 
-@[to_additive] -- TODO: make this a scoped instance in the `Pointwise` namespace
```

Comment:
I don't think you made this a scoped instance. Oversight, or is there a good reason not to do it?

Reply:
Oversight. I'll do it now.